scripts/src/main/resources/scripts/environment-project.bat

```diff
@@ -9,17 +9,22 @@ if "%WORKSPACE%" == "" (
   set WORKSPACE=%MAIN_BRANCH%
 )
 
-if "%SETTINGS_PATH%" == "" (
-  set SETTINGS_PATH=%WORKSPACES_PATH%\%MAIN_BRANCH%\%SETTINGS_REL_PATH%
+if "%WORKSPACE_VS%" == "" (
+  set WORKSPACE_VS=%WORKSPACESVS_PATH%
 )
+
+set WORKSPACE_PLUGINS_PATH=%WORKSPACES_PATH%\%WORKSPACE%\.metadata\.plugins
+set SETTINGS_PATH=%WORKSPACES_PATH%\%MAIN_BRANCH%\%SETTINGS_REL_PATH%
+set ECLIPSE_TEMPLATES_PATH=%SETTINGS_PATH%\%ECLIPSE_TEMPLATES_REL_PATH%
+rem absolute workspace path
 set WORKSPACE_PATH=%CD%\%WORKSPACES_PATH%\%WORKSPACE%
 
 if exist "%SETTINGS_PATH%\ide-properties.bat" (
   call "%SETTINGS_PATH%\ide-properties.bat"
 )
 
-set OASP4J_IDE_VERSION=${project.version}
-set IDE_CONFIGURATOR=devon-ide-configurator-${devon-ide.version}.jar
+set OASP4J_IDE_VERSION=1.4.0-SNAPSHOT
+set ECLIPSE_CONFIGURATOR=oasp4j-ide-eclipse-configurator-1.4.0-SNAPSHOT.jar
```

Comment:
Same as above in the linux version. `OASP4J_IDE_VERSION` may indeed be removed.
`IDE_CONFIGURATOR` needs to remain.